src/System.Runtime.Caching/src/System.Runtime.Caching.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyName>System.Runtime.Caching</AssemblyName>
```

Comment:
if we use the name S.R.C the result will have the same assembly identity as desktop. That is fine so long as we do'nt plan to add any API to it. If we do, we won't be able to evolve desktop separately. if we want more flexibility, we can pick a new name. That is more common.